From 012085b5a7c08730b7592986fc038d259aeb8fb1 Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Thu, 21 Apr 2022 07:54:46 +0000
Subject: [PATCH] libstdc++: make CredentialLeakDialogController destructor
 public

std::unique_ptr in credential_leak_dialog_factory::CreateController()
calls destructor of CredentialLeakDialogController, but it is
protected.
---
 chrome/browser/ui/passwords/credential_leak_dialog_controller.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/chrome/browser/ui/passwords/credential_leak_dialog_controller.h b/chrome/browser/ui/passwords/credential_leak_dialog_controller.h
index 8c4c34a..28034a9 100644
--- a/chrome/browser/ui/passwords/credential_leak_dialog_controller.h
+++ b/chrome/browser/ui/passwords/credential_leak_dialog_controller.h
@@ -46,7 +46,6 @@ class CredentialLeakDialogController : public PasswordBaseDialogController {
   // Checks whether the dialog should show cancel button.
   virtual bool ShouldShowCancelButton() const = 0;
 
- protected:
   ~CredentialLeakDialogController() override = default;
 };
 
-- 
2.35.1

